website/docs/Installation/linux-install.md

```diff
@@ -1,64 +1,131 @@
 ---
 sidebar_position: 5
+title: Linux
+description: You can install Podman Desktop on Linux from Flathub, a Flatpak bundle, or a ZIP archive.
+tags: [podman-desktop, installing, linux, flathub, flatpak]
+keywords: [podman desktop, podman, containers, installing, installation, linux, flathub, flatpak]
 ---
 
-# Linux
+# Installing Podman Desktop on Linux
 
-Check out the [Downloads](/downloads/linux) section of this website to download either the [.flatpak](https://flatpak.org/setup/Fedora) file or the zip file depending on your preference.
+You can install Podman Desktop from:
 
-To use Podman Desktop, the latest version of Podman is required. Podman is a rootless, daemonless container engine. Read more about it [here](https://podman.io/whatis.html).
+* [Flathub](#flathub)
+* [A Flatpak bundle](#flatpak-bundle)
+* [An archive](#archive)
```

Comment:
`An archive` seems odd and makes it sound like it's an afterthought? maybe "GitHub release page" would be better? I've seen that more often in community projects rather than describing it as an archive.

Reply:
Technically it's TAR.GZ file.

Old docs mentionned a ZIP file but they are all for macOS.

Reply:
Yup, but we reference a "Hub" such as Flathub, and a bundle (which is incorrect, should be Flatpak app).

It makes sense to say GitHub release, rather than archive.

Reply:
not sure to understand 'Github release' for the archive as on Github Release I can find tons of artifacts (including the .tgz and the .flatpak )

If we specify Github Release as a download area (compare to Flathub.org) then it means both the flatpak and the .tgz should be subsection of Github release.

Reply:
```suggestion
* [A compressed tarball](#archive)
```